bindinfo/handle.go

```diff
@@ -808,6 +812,7 @@ func (h *BindHandle) getRunningDuration(sctx sessionctx.Context, db, sql string,
 		return time.Since(startTime), nil
 	case <-timer.C:
 		cancelFunc()
+		logutil.BgLogger().Warn("plan verification timed out", zap.Duration("timeElapsed", time.Since(startTime)))
```

Comment:
How about also printing the normalized SQL here?

Reply:
Normalized sounds good. I thought about printing the query, but declined because it could expose sensitive data.

Reply:
In the event that a plan verification times out, there will be two warnings written to the log:
1. This line
2. The line about rejection, which will show the `verifyPlanTime` as `-1`

I've added this suggestion, but to reduce duplication I just write the digest to the second line. In the event that the statement is within 2x of the `currentPlanTime` only the second line here is printed to the log file:

```
[2020/10/19 22:17:58.819 -06:00] [WARN] [handle.go:885] ["new plan rejected"] [currentPlanTime=1.204462ms] [verifyPlanTime=1.176141ms] [digestText="select * from test . t1 where b = ? and c = ?"]
```

Reply:
In the event that a plan verification times out, there will be two warnings written to the log:
1. This line
2. The line about rejection, which will show the `verifyPlanTime` as `-1`

I've added this suggestion, but to reduce duplication I just write the digest to the second line. In the event that the statement is within 2x of the `currentPlanTime` only the second line here is printed to the log file:

```
[2020/10/19 22:17:58.819 -06:00] [WARN] [handle.go:885] ["new plan rejected"] [currentPlanTime=1.204462ms] [verifyPlanTime=1.176141ms] [digestText="select * from test . t1 where b = ? and c = ?"]
```